Index: Code/src/mainMenu/MainMenuControl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package mainMenu;ackage mainMenu;\r\n\r\npublic class MainMenuControl {\r\nprivate MainMenuView view;\r\npublic MainMenuControl(MainMenuView view) {\r\n\r\n}\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Code/src/mainMenu/MainMenuControl.java b/Code/src/mainMenu/MainMenuControl.java
--- a/Code/src/mainMenu/MainMenuControl.java	(revision b33f10568d71ece71a0616d26412e8fd2b085154)
+++ b/Code/src/mainMenu/MainMenuControl.java	(date 1739262327053)
@@ -1,4 +1,4 @@
-package mainMenu;ackage mainMenu;
+package mainMenu;
 
 public class MainMenuControl {
 private MainMenuView view;
Index: .idea/Enlingo-ITP-Project.iml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/Enlingo-ITP-Project.iml b/.idea/Enlingo-ITP-Project.iml
new file mode 100644
--- /dev/null	(date 1739261897386)
+++ b/.idea/Enlingo-ITP-Project.iml	(date 1739261897386)
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$">
+      <sourceFolder url="file://$MODULE_DIR$/Code/src" isTestSource="false" />
+    </content>
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
\ No newline at end of file
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
new file mode 100644
--- /dev/null	(date 1739262314144)
+++ b/.idea/misc.xml	(date 1739262314144)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_19" default="true" project-jdk-name="corretto-19" project-jdk-type="JavaSDK">
+    <output url="file://$PROJECT_DIR$/out" />
+  </component>
+</project>
\ No newline at end of file
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1739259197269)
+++ b/.gitignore	(date 1739259197269)
@@ -0,0 +1,1 @@
+/non-code/Aktivitätsdiagramm/AktivitätsDiagramm_Enlingo.asta.lock
Index: .idea/modules.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/modules.xml b/.idea/modules.xml
new file mode 100644
--- /dev/null	(date 1739259197243)
+++ b/.idea/modules.xml	(date 1739259197243)
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="ProjectModuleManager">
+    <modules>
+      <module fileurl="file://$PROJECT_DIR$/.idea/Enlingo-ITP-Project.iml" filepath="$PROJECT_DIR$/.idea/Enlingo-ITP-Project.iml" />
+      <module fileurl="file://$PROJECT_DIR$/Wordle/Wordle.iml" filepath="$PROJECT_DIR$/Wordle/Wordle.iml" />
+    </modules>
+  </component>
+</project>
\ No newline at end of file
Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/vcs.xml b/.idea/vcs.xml
new file mode 100644
--- /dev/null	(date 1739259197252)
+++ b/.idea/vcs.xml	(date 1739259197252)
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1739259197279)
+++ b/.idea/.gitignore	(date 1739259197279)
@@ -0,0 +1,8 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
+# Datasource local storage ignored files
+/dataSources/
+/dataSources.local.xml
